ros/src/driveworks/packages/autoware_driveworks_gmsl_interface/CMakeLists.txt

```diff
@@ -0,0 +1,157 @@
+# Copyright (c) 2016, NVIDIA CORPORATION.  All rights reserved.
+
+cmake_minimum_required(VERSION 2.8.3)
+project(gmsl_camera)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-error -Wno-deprecated-declarations -std=c++11")
+
+##################
+## ROS packages ##
+##################
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS
+  roscpp
+  sensor_msgs
+  std_msgs
+  cv_bridge
+  image_transport
+  camera_info_manager
+  nodelet
+)
+
+
+##############################
+## Set basic configurations ##
+##############################
+set(CMAKE_MODULE_PATH /usr/local/driveworks/samples/cmake)
```

Comment:
From PR #1447, the environment variables are:

export DRIVEWORKS_TOOLKIT_ROOT_DIR=/usr/local/driveworks
export CUDA_TOOLKIT_ROOT_DIR=/usr/local/cuda
export NVIDIA_TENSORRT_ROOT_DIR=/usr/local/nvidia/tensorrt
export NVIDIA_DRIVE_SDK_ROOT_DIR=$HOME/NVIDIA/Drive/5050bL_SDK/DriveSDK/drive-t186ref-linux

Should we create a FindDriveWorks.cmake to automate this?

Reply:
> Should we create a FindDriveWorks.cmake to automate this?

That's a good idea, the variables in #1447 are a bit obscure right now.